native-sql-engine/core/pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>3.6.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-auth</artifactId>
+      <version>3.2.0</version>
+    </dependency>
```

Comment:
let's follow the new buulding profile and use hadoop.version
https://github.com/oap-project/native-sql-engine/pull/277/files

Reply:
fixed in this pr